docs/source/tutorial/cli/fingerprint.rst

```diff
@@ -0,0 +1,36 @@
+.. _fingerprint:
+
+Fingerprint
+============================
+
+To calculate the learned representations (encodings) of model inputs from a pretrained model, run
+
+.. code-block::
+   
+   chemprop fingerprint --test-path <test_path> --model-path <model_path> 
+
+where :code:`<test_path>` is the path to the CSV file containing SMILES strings, and :code:`<model_path>` is the path to the trained model (.ckpt or .pt file) or a directory containing the trained model files. By default, predictions will be saved to the same directory as the test path. If desired, a different directory can be specified by using :code:`--output <path>`
+
+For example:
+
+.. code-block::
+  
+    chemprop predict --test-path tests/data/smis.csv \
+        --model-path tests/data/example_model_v2_regression_mol.ckpt \
+        --output fps.csv
```

Comment:
```suggestion
    chemprop fingerprint --test-path tests/data/smis.csv \
    --model-path tests/data/example_model_v2_regression_mol.ckpt \
    --output fps.csv
```

Reply:
I am trying to make this consistent with what we used in the training documentation. If you think it makes sense, please also change the one in the prediction documentation.